draft-ietf-quic-tls.md

```diff
@@ -1168,89 +1168,226 @@ anticipation of receiving a ClientHello.
 
 # Key Update
 
-Once the handshake is confirmed, it is possible to update the keys. The
-KEY_PHASE bit in the short header is used to indicate whether key updates
-have occurred. The KEY_PHASE bit is initially set to 0 and then inverted
-with each key update.
+Once the handshake is confirmed (see {{handshake-confirmed}}), it is possible to
+update the keys used to protect packets.
```

Comment:
We might want to change "possible to update the keys" to "possible to initiate a key update"? This is because an endpoint might receive a key update(s) before it confirms the handshake.

Or, we can punt such a change to the PR that will fix the handshake key discard problem.